src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -348,7 +348,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 CollectTupleFieldMemberNames(name, i + 1, numElements, elementNames)
 
-                Dim boundArgument As BoundExpression = BindRValue(argumentSyntax.Expression, diagnostics)
+                Dim boundArgument As BoundExpression = BindValue(argumentSyntax.Expression, diagnostics)
```

Comment:
I didn't understand this change. Shouldn't the expressions composing a tuple literal be RValues?